js/ad_manager_vast.js

```diff
@@ -294,6 +347,22 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+     * Called when the ad starts playback.
+     * @public
+     * @method Vast#adVideoPlaying
+     */
+    this.adVideoPlaying = function() {
+      if (currentAd && currentAd.ad.nextAdInPod) {
+        var metadata = currentAd.ad.nextAdInPod;
+        var ad = generateAd(metadata);
+
+        if (metadata.streamUrl != null || (ad.adType == this.amc.ADTYPE.LINEAR_VIDEO && !_.isEmpty(metadata.streams))) {
```

Comment:
streamUrl is used for nonlinear ads and streams is used for linear ads. This is the convention that was used prior to my changes.